.github/workflows/create-prerelease-branch.yml

```diff
@@ -0,0 +1,126 @@
+# Bump package version numbers and bundle changelogs into changelog file
+name: Create pre-release branch
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Branch or tag to create pre-release from'
+        required: true
+        default: 'main'
+      bump_type:
+        description: 'Bump type'
+        required: true
+        type: choice
+        options:
+          - stable minor
+          - stable patch
+          - beta
+
+jobs:
+  create_pre-release:
+    name: Bump versions and make changelog
+    runs_on: ubuntu-latest
+    env:
+      MAIN_BRANCH: 'main'
+    steps:
+      # Check-out repo
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          # Use a machine account when checking out. This is to workaround the issue were GitHub
+          # actions, when using the default account, cannot trigger other actions. And we want this
+          # action to trigger the regular CI pipeline on the created branch.
+          # This machine account is only for this PAT, pwd was created and thrown away
+          # If any update needed, create a new account, add access to the repo and generate a new PAT
+          token: ${{ secrets.MACHINE_ACCOUNT_PAT }}
+
+      # Setup bot information for creating pull request
+      # Here we use the id from the github actions bot: https://api.github.com/users/better-informatics%5Bbot%5D
+      - name: Setup bot git information
+        run: |
+          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
+          git config user.name "github-actions[bot]"
+
+      # Check out onto desired branch or tag to create release from
+      - name: Checkout tag/branch
+        run: git checkout ${{ github.event.inputs.branch }}
+
+      # Ensure node version is great enough
+      - name: Use Node.js v14.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: '14.x'
+
+      # Try get node_modules from cache
+      - name: Restore node_modules from cache
+        uses: actions/cache@v2
+        with:
+          path: common/temp/pnpm-store
+          key: ${{ runner.os }}-${{ hashFiles('common/config/rush/pnpm-lock.yaml') }}
+
+      # Bump package versions
+      - name: Bump package versions
+        run: node common/scripts/bump-beta-version.js
+
+      # Generate changelog using beachball
+      - name: Generate changelog using beachball
+        run: node common/scripts/beachball-changelog-only.js
+
+      - name: Synchronize package version reported to telemetry
+        run: node common/scripts/sync-telemetry-package-version
+
+      # Important to check version consistency again after bumping versions.
+      - name: Ensure all package versions are consistent
+        run: rush ensure-consistent-versions
```

Comment:
I will update #1687 so that `ensure-consistent-versions` doesn't check telemetry version anymore. You can keep this check here so that we error out sooner in case of a mismatch.